packages/common-config/hardhat.config.ts

```diff
@@ -12,8 +12,6 @@ import path from 'node:path';
 
 // Load common .env file from root
 dotenv.config({ path: path.resolve(__dirname, '..', '..', '.env') });
-// Load common .env file at ./packages/config-common/.env
-dotenv.config({ path: path.resolve(__dirname, '.env') });
```

Comment:
Remove the option to use the `./packages/config-common/.env` file

Reply:
:pray: